downstream/modules/hub/proc-create-synclist.adoc

```diff
@@ -33,11 +33,28 @@ The following domain names must be in the allow list:
 ** `ansible-galaxy.s3.amazonaws.com`
 * SSL inspection is disabled either when using self signed certificates or for the Red Hat domains.
 
+[IMPORTANT]
+
+====
+
+Before you begin your content sync, consult our Knowledgebase article link:https://access.redhat.com/articles/7118757[Resource requirements for syncing automation content] to ensure that you have the resources to sync the collections you need. 
```

Comment:
Not sure how strict our rules are about cross-reference language, but should "...consult our" be "...see the"?